src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1AssessmentEmailServiceTest.kt

```diff
@@ -195,7 +195,7 @@ class Cas1AssessmentEmailServiceTest {
 
     @Test
     fun `assessmentAllocated sends an email to a user if they have an email address and an emergency assessment with a deadline of today`() {
-      val deadline = OffsetDateTime.now().minusHours(2)
+      val deadline = OffsetDateTime.now()
```

Comment:
That's one benefit of coding in the early hours :D 

I looked into why we did this. I think originally this test was part of the deadline calculation logic so we had to force a deadline that has past. Because this just tests the email part of of allocations, we no longer need the minusHours. Good spot!